bundles/org.openhab.binding.network/src/main/resources/OH-INF/i18n/network.properties

```
@@ -60,37 +60,37 @@ thing-type.config.network.servicedevice.port.description = The port on which the
 thing-type.config.network.servicedevice.refreshInterval.label = Refresh Interval
 thing-type.config.network.servicedevice.refreshInterval.description = States how long to wait after a device state update before the next refresh shall occur (in ms)
 thing-type.config.network.servicedevice.retry.label = Retry
-thing-type.config.network.servicedevice.retry.description = Defines how many times a connection attempt shall occur, before the device is stated as offline
+thing-type.config.network.servicedevice.retry.description = Defines how many times a connection attempt shall occur before the device is stated as offline.
 thing-type.config.network.servicedevice.timeout.label = Timeout
-thing-type.config.network.servicedevice.timeout.description = States how long to wait for a response (in ms), before if a device is stated as offline
+thing-type.config.network.servicedevice.timeout.description = States how long to wait for a response (in ms) before a device is stated as offline.
 thing-type.config.network.speedtest.fileName.label = File Name
-thing-type.config.network.speedtest.fileName.description = Name of the file to download from test server
+thing-type.config.network.speedtest.fileName.description = Name of the file to download from test server.
 thing-type.config.network.speedtest.initialDelay.label = Initial Delay
 thing-type.config.network.speedtest.initialDelay.description = Delay before starting the first speed test (minutes) after initialization of the binding.
 thing-type.config.network.speedtest.maxTimeout.label = Timeouts
-thing-type.config.network.speedtest.maxTimeout.description = Number of timeout that can happend before the device is stated as offline
+thing-type.config.network.speedtest.maxTimeout.description = Number of timeout that can happen before the device is stated as offline.
```

Since it was changed in the XML:
```suggestion
thing-type.config.network.speedtest.maxTimeout.description = Number of timeouts that can happen before the device is stated as offline.
```